shopfloor/services/cluster_picking.py

```diff
@@ -456,6 +456,11 @@ def scan_line(self, picking_batch_id, move_line_id, barcode):
         if product and move_line.product_id == product:
             return self._scan_line_by_product(picking, move_line, product)
 
+        if not product:
+            packaging = search.packaging_from_scan(barcode)
```

Comment:
Can we have a packaging without a package (search for on line 450) because if the answer is no this will line will never be reached !

Reply:
yes, that's the product packaging, like the barcode on the box